index.html

```diff
@@ -1398,17 +1404,24 @@ <h2>String Matching of Syntactic Content in Document Formats and Protocols</h2>
       <section id="specifying-content-restrictions">
 		  <h2>Specifying Content Restrictions</h2>
 		  
-		  <p>One of the ways in which string matching can be made more effective and consistent is by applying restrictions to the content that is to be matched. Document formats and protocols often do this by defining a namespace or set of rules for what is legal in a given context.</p>
+		  <p>One of the ways in which string matching can be made more effective and consistent is by applying restrictions to the content that is to be matched. When building a <a>vocabulary</a> and especially when defining <a>user-supplied values</a> within that vocabulary, specifications for document formats and protocols often define a namespace or set of rules for what is legal in a given context. This usually includes length and content restrictions.</p>
+		  
+		  <p class="requirement"><span class="qrec">[S]</span> Specifications SHOULD NOT allow the <code>C0</code> (<span class=uname>U+0000</span> to <span class=uname>U+001F</span>) and <code>C1</code> (<span class=uname>U+0080</span> to <span class=uname>U+009F</span>) control characters.</p>
```

Comment:
What is the context of this guidance? I think in context like HTML [phrasing content](https://html.spec.whatwg.org/multipage/dom.html#phrasing-content), control characters *should* be allowed. If it refers to identifiers, we should mention that.

Another question: should we allow surrogates and [noncharacters](https://unicode.org/reports/tr44/#Noncharacter_Code_Point)?

Reply:
It's in the context of identifiers. I can clarify.

Unpaired surrogates and non-character code points are mostly handled by data types such as DOMString or definitions in the Infra spec. It's not wrong to ban them here, though. You heard us struggling with describing the rules in the teleconference. Anyway, point taken: more work is needed here.